content/tutorial-plotting-fractals.md

```diff
@@ -0,0 +1,553 @@
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.4
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Plotting Fractals
+
++++
+
+![Fractal picture](tutorial-plotting-fractals/fractal.png)
+
++++
+
+Fractals are beautiful, compelling mathematical forms that can be oftentimes created from a relatively simple set of instructions. In nature they can be found in various places, such as coastlines, seashells, and ferns, and even were used in creating certain types of antennas. The mathematical idea of fractals was known for quite some time, but they really began to be truly appreciated in the 1970's as advancements in computer graphics and some accidental discoveries lead researchers like Mandelbrot to stumble upon the truly mystifying visualizations that fractals possess. 
+
+Today we will learn how to plot these beautiful visualizations and will start to do a bit of exploring for ourselves as we gain familiarity of the mathematics behind fractals and will use the ever powerful NumPy universal functions to perform the necessary calculations efficiently.
+
++++
+
+## What you'll do
+
+- Write a function for computing Mandelbrot fractals
+- Write a function that computes Newton fractals 
+- Experiment with variations of general fractal types
+
++++
+
+## What you'll learn
+
+- A better intuition for how fractals work mathematically 
+- A basic understanding about NumPy Universal Functions and Boolean Indexing
+- The basics of working with complex numbers in NumPy 
+- How to create your own unique fractal visualizations
+
++++
+
+## What you'll need
+
+- [Matplotlib](https://matplotlib.org/)
+- make_axis_locatable function from mpl_toolkits API
+ 
+which can be imported as follows:
+
+```{code-cell} ipython3
+import numpy as np
+import matplotlib.pyplot as plt
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+```
+
+- Some familiarity with Python, NumPy and matplotlib
+- An idea of elementary mathematical functions, such as [exponents](https://en.wikipedia.org/wiki/Exponential_function), [sin](https://en.wikipedia.org/wiki/Sine), [polynomials](https://en.wikipedia.org/wiki/Polynomial) etc
+- A very basic understanding of [complex numbers](https://en.wikipedia.org/wiki/Complex_number) would be useful
+- Knowledge of [derivatives](https://en.wikipedia.org/wiki/Derivative) may be helpful
+
++++
+
+## Warmup
+
+To gain some intuition for what fractals are, we will begin with an example.
+
+Consider the following equation: 
+
+$f(z) = z^2 -1 $ 
+
+where z is a complex number (i.e of the form $a + bi$ )
+
+For our convenience, we will write a Python function for it
+
+```{code-cell} ipython3
+def f(z):
+    return np.square(z) - 1  
+```
+
+Note that the square function we used is an example of a **[NumPy Universal Function](https://numpy.org/doc/stable/reference/ufuncs.html)**; we will come back to the significance of this decision shortly. 
```

Comment:
We have [a PR up](https://github.com/numpy/numpy/pull/19516) to update this doc - so we might need to change this link in the future. Also - @rossbar do you think it would be better to use intersphinx for the tutorials? This had escaped me before but now I'm starting to think it would be the best solution.

Reply:
> Also - @rossbar do you think it would be better to use intersphinx for the tutorials? This had escaped me before but now I'm starting to think it would be the best solution.

The reason we decided not to use `intersphinx` links in the tutorials themselves is because they will only render in the html output, not in the notebooks (e.g. for binder users). I haven't been paying super close attention to when/whether the browser-based interfaces will support MyST syntax, but there seems to be promising work in support of that general direction, e.g. executablebooks/meta#449. 

I don't think we have any data on the number of binder users (do we?) so I think it's probably best to stick with commonmark compliant markdown for now so that the links work for all users. If/when Jupyter supports MyST in the browser, it'd be a great mini-sprint activity to go through and "myst-ify" all the notebooks to take advantage of all the awesome additional features :)

Of course, using intersphinx to link **to** the tutorials (e.g. from the official numpy docs) is still possible & encouraged (e.g. once this is merged it might be nice to add a link to the Mandelbrot example in the Quickstart tutorial!)